index.html

```diff
@@ -1 +1,11 @@
-hello world
+!doctype <!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
what does this mean - `dir="ltr"`

Reply:
Reply like this (below!)

Reply:
HERE!!!

Reply:
lang stands for language which is English in this case.
dir stands for text direction which is left to right in this case.